src/server/session.ts

```diff
@@ -299,9 +299,10 @@ export function formatDiagnosticToProtocol(diag: Diagnostic, includeFileName: bo
         : common;
 }
 
-export interface PendingErrorCheck {
+interface PendingErrorCheck {

```

Comment:
I have no idea why we were exporting this in the first place, since it's only used as a parameter type of a private method. But I can export it again if it's relevant.